lib/sizzle.patched.js

```diff
@@ -1488,6 +1491,20 @@ Expr = Sizzle.selectors = {
 
 		"contains": markFunction(function( text ) {
 			text = text.replace( runescape, funescape );
+
+			// Patch #5: Support regular expressions as arguments of :contains pseudo class.
+			if(/^\s*\/.*\/\s*$/.test(text)) {
+				try {
+					var regex = new RegExp(text.trim().slice(1, -1));
+				} catch(e) {
+					Sizzle.error(text); // the expression is invalid
```

Comment:
Sizzle.error throws an exception. I have chosen this because it was how Sizzle throws error in its other parts.